notebooks/scripts/check_edit_location.py

```diff
@@ -0,0 +1,248 @@
+    logger = initialize_logging(arguments.log_file)
+    # Create the GIS
+    logger.info("Authenticating...")
+    # First step is to get authenticate and get a valid token
+    gis = GIS(arguments.org_url,
+              username=arguments.username,
+              password=arguments.password,
+              verify_cert=not arguments.skip_ssl_verification)
+
+    # Get the feature layer
+    logger.info("Getting feature layer")
+    if arguments.project_id and arguments.layer_url:
+        logger.info("Please use either project id or layer url as your work order layer, not both")
+        sys.exit(0)
+    elif arguments.project_id:
+        item = gis.content.get(arguments.project_id)
+        layer = workforce.Project(item).assignments_layer
+    elif arguments.layer_url:
+        layer = FeatureLayer(arguments.layer_url)
+    else:
+        logger.info("Please provide either a portal id for your Workforce project or a feature service URL for your survey/collector layer")
+        sys.exit(0)
+    try:
+        tracks_layer = gis.admin.location_tracking.tracks_layer
+    except Exception as e:
+        logger.info(e)
+        logger.info("Getting location tracking service failed - check that you are an admin and that location tracking is enabled for your organization")
+        sys.exit(0)
+        
+    # Return invalid work orders
+    workers = arguments.workers.split(",")
+    invalid_work_orders = get_invalid_work_orders(layer,
+                                                  arguments.field_name,
+                                                  arguments.time_tolerance,
+                                                  arguments.distance_tolerance,
+                                                  arguments.min_accuracy,
+                                                  workers,
+                                                  tracks_layer,
+                                                  logger)
+    if len(invalid_work_orders) == 0:
+        logger.info("No features found that match the criteria you've set")
+    else:
+        for work_order in invalid_work_orders:
+            logger.info("The user who last edited the feature with OBJECTID {} was not within the desired accuracy for the field you are checking".format(work_order.attributes[return_field_name(layer, "objectid")]))
+
+    
+if __name__ == "__main__":
+    # Get all of the commandline arguments
+    parser = argparse.ArgumentParser("Check that the worker was nearby when editing features")
+    parser.add_argument('-u', dest='username', help="The username to authenticate with", required=True)
+    parser.add_argument('-p', dest='password', help="The password to authenticate with", required=True)
+    parser.add_argument('-org', dest='org_url', help="The url of the org/portal to use", required=True)
+    # Parameters for tracker
+    parser.add_argument('-workers', dest='workers', help="Comma separated list of user_id's for the workers to check")
```

Comment:
I saw that the `+` syntax was used in the original script, but I took it out because I think it makes the arguments look separate as opposed to what it is - passing multiple values into one arg - and it makes putting the data into a list a bit trickier. To me, comma separated seems like a standard way of inserting data

Reply:
Doesn't it default to putting it into a list? (e.g `args.workers`)

Reply:
Should have clarified my above comment more - yes, it does default to putting it in a list, but one thing I've seen recently from people on our team trying to use the scripts for the first time is that they might put their arg in quotation marks. I am trying to handle quotation marks for arguments as well. So if my workers are:

`-workers "apulver_tracker1 jmcmanus_tracker1"` - the `+` syntax will create a list with one element

while as comma separated
`-workers "apulver_tracker1,jmcmanus_tracker1" ` will correctly parse the string into a list of two elements

Reply:
Ah okay that makes sense. Let's make sure we strip whitespace then, cuz I know I'll do "apulver, jmcmanus" and it won't work